### PR TITLE
Adding fquery test case for nested object #223

### PR DESCRIPTION
**Pull request by btkatoutkm**
Please review #223, and merge to master branch.

### CODE REVIEW

**Comment by kazuki43zoo** [comment]
Hi @btkatoutkm, please confirm following comments:

- [ ] Please divide test case method per test case scenarios (but existing test cases are out of focus).
- [ ] Please add field category name, field name and field description for enable inferring input value in JSP, if needed.
- [ ] Query_Display(f:query(MapOfJavaBean)) is mismatch with actual test case. In actually, it use "Map of Simple Value" (= not used "Map of Java Bean"). Please keep the consistency with test case scenario and test application.

File: terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/el/ElTest.java [hunk 1]
```diff
@@ -403,6 +403,272 @@ public void test06_Query_Display() {
     }
 
     @Test
+    public void test06_09_NestedJavaBean() {
+        driver.get(applicationContextUrl);
+        driver.findElement(By.id("EL")).click();
```
Comment (kazuki43zoo) [inline terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/el/ElTest.java]: Probably, two step of above are unnecessary. it perform by `@Before` method of parent class.

File: terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/el/ElTest.java [hunk 2]
```diff
@@ -403,6 +403,272 @@ public void test06_Query_Display() {
...
+                        .getAttribute("value"), is("yamada"));
+        assertThat(
+                driver.findElement(By.id("criteria.age")).getAttribute("value"),
+                is("20"));
+        assertThat(
+                driver.findElement(By.id("rememberCriteria")).getAttribute(
+                        "value"), is("true"));
+
+        // screen capture
+        screenCapture.save(driver);
```
Comment (kazuki43zoo) [inline terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/el/ElTest.java]: Screen capture is taken automatically at the finish of test.
Therefore this step is unnecessary.

File: terasoluna-gfw-functionaltest-web/src/main/webapp/WEB-INF/views/el/dateTimeFormatQueryOutput.jsp [hunk 3]
```diff
@@ -0,0 +1,45 @@
+
+<h2>Query Display f:query(DateTimeFormat)</h2>
+
+<span>Input Data</span>
+<form:form action="${pageContext.request.contextPath}/el/6_13/search"
+	method="GET" modelAttribute="dateForm5">
+	<fieldset>
+		<legend>Nested Bean(Parent)</legend>
+		<p>
+			<span>Date</span>
```
Comment (kazuki43zoo) [inline terasoluna-gfw-functionaltest-web/src/main/webapp/WEB-INF/views/el/dateTimeFormatQueryOutput.jsp]: Please add description for allowed date format. 

e.g.) yyyy-MM-dd

**Comment by kazuki43zoo** [comment]
Hi @btkatoutkm,please add the following test case (for bug fix):

* Set an any default value(=not null) in Form object using `@ModelAttribute` method
* Override to blank value(null or empty is binding) value in HTML FORM and submit it.
* Perform pagination and assert HTML FROM status (all fields are null or empty)

In this test case, please use the `StringTrimmingEditor`.

**Comment by btshimizukza** [comment]
@btkatoutkm 

This PR is OK!
Please summarize to the one commit because i will merged this PR.

**Pull request closed without merging**